### ISSUE TITLE
Should Move `local_path` in class Opinion to OpinionCluster

### ISSUE THREAD

**Issue by flooie** [freelawproject/courtlistener#2955]
As it stands now - we store the Columbia file path location on the Opinion object, but that is incorrect.  

The data in that file is for the opinion cluster - and we duplicate that location if an opinion cluster has multiple opinions.  We should default to the better system we set up for `filepath_json_harvard` on the opinion cluster and create a new field called `filepath_xml_columbia` 

I would suggest we repopulate the field when we update ordering for opinions in an upcoming PR.

**Comment by mlissner** [comment]
Seems like a good idea to me. Would it make sense to rename `filepath_json_harvard` as `source_data` and use it for columbia too, so we don't slowly amass fields?

**Comment by mlissner** [comment]
I could also see it being a good solution to add a source table that can be FK'ed to objects. Hm.

**Comment by flooie** [comment]
Thats fine with me - but seems like more work then just slapping on one more- but perhaps its the right approach.   Especially if this gets expanded to more countries... or sources.

**Comment by mlissner** [comment]
Yeah, as we add more sources, a joined table feels logical to me. It's also a lot easier to drop columns than add them to these huge tables, so that motivates me as well.

**Comment by flooie** [comment]
Normally I would agree with you - but once I started moving things around - I realized how much opinion local path is used to save and store PDF files we've scraped.  But I still think it should be moved to the cluster.  

I think it requires also moving download_url to the new OpinionFile table or whatever we end up calling it and just a slight refactoring of the saving method in a couple places across the code base.  @mlissner

**Comment by mlissner** [comment]
You say: 

>  I still think it should be moved to the cluster.

And: 

> it requires also moving download_url to the new OpinionFile table 

So I'm confused, do you agree about creating the new table or do you think moving columns around is better? 

Either way, have you verified that none of the clusters with multiple opinions have multiple items in these columns? E.g., this would be a problem, right:

```
Cluster1:
 - Opinion1
     - download_url: Some value.pdf
 - Opinion2
     - download_url: Some other value.pdf
```

I don't know how we'd migrate that to a single column on the cluster? 

```
cluster1:
 - download_url: Some value.pdf, some other value.pdf?
 - Opinion1
 - Opinion2
```

That doesn't work, right?
